Rubberduck.Core/UI/Command/MenuItems/CommandBars/RubberduckCommandBar.cs

```diff
@@ -162,8 +171,8 @@ protected virtual void Dispose(bool disposing)
             }
 
             _selectionService.SelectionChanged -= OnSelectionChange;
-            _parser.State.StateChanged -= OnParserStateChanged;
-            _parser.State.StatusMessageUpdate -= OnParserStatusMessageUpdate;
+            _state.StateChanged -= OnParserStateChanged;
```

Comment:
should be `_state.StateChangedHighPriority  -= OnParserStateChanged`, no?

Reply:
Thanks, I missed that.